Tasks/day.js

```diff
@@ -1,15 +1,11 @@
-// Get day number
-
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
-
-_parse_day_ = (s) => {
-  let i;
-  for (i = 0; i < D.length; i++) {
-    if (s.startsWith(D[i].toLowerCase())) {
-      return i + 1;
-    }
+'use strict';
+const WEEK_DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const parseDay = (string) => {
+  let i = 0;
+  for (const day of WEEK_DAYS) {
+    if (string.startsWith(day.toLowerCase())) return i + 1;
```

Comment:
```suggestion
    if (string.startsWith(day) return i + 1;
```
You can delete the function 'toLowerCase'